### PR TITLE
Add alert for broken Tahoma integration

### PR DESCRIPTION
**Pull request by iMicknl**

### CODE REVIEW

File: alerts/tahoma.markdown [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: "Tahoma integration incompatible with firmware 2020.3.4-12"
+created: 2020-07-03 12:00:00
+updated: 2020-07-03 12:00:00
+integrations:
+  - tahoma
+github_issue: https://github.com/home-assistant/core/issues/37291
+homeassistant: ">0.49"
```
Comment (iMicknl) [inline alerts/tahoma.markdown:8]: Not sure about this value...

File: alerts/tahoma.markdown [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+---
+
+Updating your Somfy TaHoma bridge 2020.3.4-12 will cause the Tahoma integration for Home Assistant to no longer work.
+
+## Workaround
+The Tahoma integration for Home Assistant is in the process of being completely rewritten and modernized. An initial beta version has been [released as a custom integration](https://github.com/imicknl/ha-tahoma) in order to gather feedback and bug reports. After the testing period concludes the changes will be integrated back into Home Assistant.
```
Comment (balloob) [inline alerts/tahoma.markdown:14]: Isn't the actual workaround to use the official Somfy integration?

File: alerts/tahoma.markdown (hunk 1 above)
Comment (balloob) [reply alerts/tahoma.markdown:8]: Should be 0.59 as that was the version the integration got introduced https://www.home-assistant.io/integrations/tahoma/
```suggestion
homeassistant: ">0.59"
```

File: alerts/tahoma.markdown (hunk 2 above)
Comment (iMicknl) [reply alerts/tahoma.markdown:14]: Unfortunately not, the official Somfy API only supports a few devices. However, would be good to mention that in this alert as well. The Somfy and new Somfy TaHoma integration could be used next to each other.

Comment (iMicknl) [reply alerts/tahoma.markdown:16]: @balloob updated the text to first steer people to the Somfy integration. Thanks!

Comment (iMicknl) [reply alerts/tahoma.markdown:16]: @tetienne @vlebourl any comments / additions?

Comment (tetienne) [reply alerts/tahoma.markdown:16]: Can we not do a hotfix directly on the Tahoma component? I don't know if your fix was complex or not.

**Pull request merged**